src/status_im2/contexts/communities/home/style.cljs

```diff
@@ -1,12 +1,13 @@
 (ns status-im2.contexts.communities.home.style
   (:require [quo2.foundations.colors :as colors]
-            [react-native.platform :as platform]))
+            [react-native.platform :as platform]
+            [react-native.reanimated :as reanimated]))
 
 (def header-height 245)
 
 (def tabs
   {:padding-horizontal 20
-   :padding-top        16
+   :padding-top        8
```

Comment:
I reduced the top space of the tabs, the explanation is in the review-notes of this PR